templates/pageLevelProgressNavigation.hbs

```diff
@@ -1,6 +1,8 @@
 {{! make the _globals object in course.json available to this template}}
 {{import_globals}}
 
+<div class="pagelevelprogress__indicator-wrapper"></div>
+
 <span class="tooltip">
```

Comment:
Is it worth removing the tooltip span at this point?

As far as I can see, _ariaLabels.progress label doesn't exist.
https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/blob/master/schema/course.schema.json

https://github.com/adaptlearning/adapt-contrib-core/blob/3668a15c1e9777c23615746cd1d8465f9d729a27/schema/course.schema.json#L142-L300

Reply:
Yep, agreed. Done.